src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -343,6 +343,10 @@ export interface SavedObjectsOpenPointInTimeOptions extends SavedObjectsBaseOpti
    * An optional ES preference value to be used for the query.
    */
   preference?: string;
+  /**
+   * An optional list of namespaces to be used by the PIT.
```

Comment:
Our separation between core / security makes documentation really hard here. But in the absence of another good place to write documentation it feels like we should maybe add some details about how this option behaves when the spaces plugin is enabled? Otherwise it's very difficult to figure out how to use this.

```suggestion
   * An optional list of namespaces to be used by the PIT.
   *
   * When the spaces plugin is enabled:
   *.  - this will default to the user's current space as determined by the URL
   *  - if namespaces is specified the user's current space will be ignored
   *  - `['*']` will search across all available spaces
```

Reply:
Yea, that's fair. I don't think of any better place...